Source/Core/VideoBackends/Vulkan/Renderer.cpp

```diff
@@ -153,14 +162,16 @@ void Renderer::BeginFrame()
 
 void Renderer::ClearScreen(const EFBRectangle& rc, bool colorEnable, bool alphaEnable, bool zEnable, u32 color, u32 z)
 {
-	VkClearAttachment clear_attachments[2];
-	uint32_t num_clear_attachments = 0;
+  // Native -> EFB coordinates
+  TargetRectangle target_rc = Renderer::ConvertEFBRectangle(rc);
 
 	// Clearing must occur within a render pass.
 	m_state_tracker->BeginRenderPass();
```

Comment:
It actually doesn't. vkCmdClearColorImage and vkCmdClearDepthStencilImage will work outside a renderpass and are treated as a transfer to that image. Though it will stomp on all channels in the image.